crates/ruma-identity-service-api/src/association/check_3pid_validity/v2.rs

```diff
@@ -0,0 +1,52 @@
+//! [GET /_matrix/identity/v2/3pid/getValidated3pid](https://matrix.org/docs/spec/identity_service/r0.3.0#get-matrix-identity-v2-3pid-getvalidated3pid)
+
+use js_int::UInt;
+use ruma_api::ruma_api;
+
+ruma_api! {
+    metadata: {
+        description: "Determines if a given 3pid has been validated by a user.",
+        method: GET,
+        name: "check_3pid_validity",
+        path: "/_matrix/identity/v2/3pid/getValidated3pid/",
+        rate_limited: false,
+        authentication: AccessToken,
+    }
+
+    request: {
+        /// The Session ID enerated by the `requestToken` call.
+        #[ruma_api(query)]
+        pub sid: String,
+
+        /// The client secret passed to the `requestToken` call.
+        #[ruma_api(query)]
+        pub client_secret: String,
+
+    }
+
+    response: {
+        /// The medium type of the 3pid.
```

Comment:
The spec is inconsistent with capitalization of "3pid", but I'd prefer to capitalize it since it's an acronym. (Also below.)
```suggestion
        /// The medium type of the 3PID.
```